### ISSUE TITLE
PHP 7.3 compatibility

### ISSUE THREAD

**Issue by holsandre** [mrmap-community/Mapbender2.8#42]
# Status Quo #

The Mapbender2.8 Project is compatible and tested on PHP 7.0.

# Improvement # 
Make Mapbender2.8 PHP 7.3 compatible.

**Comment by holsandre** [comment]
Sven:
I created this Issue as a reminder. There is a branch for this issue already, but seemingly nobody had time to test it yet.
https://git.osgeo.org/gitea/GDI-RP/Mapbender2.8/src/branch/mapbender_php_7.3

**Comment by holsandre** [comment]
We have a test system running on the php 7.3 branch, unfortunately this is only internal atm.

Things that do NOT work right now:

    PDF Print
    Saving/loading wmc’s

Maybe ill come up with more in the future.

**Comment by holsandre** [comment]
simonseyock:


I created a new branch on this repository where i cherry-picked the PHP 7.3 patch that @hollsandre mentioned. I suppose that patch was based on work of me that I’ve done earlier.

I included updates of PHP functions also in the extensions where the branch was missing some functions. I think this was the cause of the PDF print not working. The WMC saving/loading also seems to work now.
